pilz_psenscan_test_area/automatic_acceptance_test_psen_scan.sh

```diff
@@ -2,13 +2,16 @@
 
 [[ -z $TEST_AREA_SCRIPTS_PATH ]] && { echo "Environment variable TEST_AREA_SCRIPTS_PATH is not set! Cannot continue!" >&2; exit 1; }
 [[ -z $1 ]] && export TARGET_BRANCH="melodic-devel" || export TARGET_BRANCH="$1"
+[[ -z $2 ]] || export TEST_DURATION="$2"
```

Comment:
Since there is no default value for `TEST_DURATION` it feels a little off to have it as the second parameter

Reply:
What do you mean by that? I thought a default value is redundant since the `.test`-file already specifies a default value.